### DISCUSSION TITLE
Can I use the pipeline/telomere code to accurately identify telomere start sites?

### DISCUSSION THREAD

**Discussion by santiago-es** [VGP/vgp-assembly#64] [Q&A]
Hi, I'm a graduate student fairly new to sequencing analysis so I apologize for what might be newbie questions:

1. can I use the pipeline/telomere code to accurately identify telomere start sites? I suspect I should be able to and I have run the pipeline on the CHM13 T2T fasta file as a test run. In the *.Windows file I have many windows, but I'm unsure if I'm able to interpret any of them as the start site for the telomere in the assembly?

2. Would it be possible to use this to determine telomere length in my own sequencing data, assuming I was able to sequence across the whole telomere? 
Thank you!

**Comment by skoren** [comment]
1. The windows output will report any 1kb window with a minimum fraction of canonical telomere sequence in it. The windows are overlapping so the output is redundant. The default is quite low (40%) but you can pass a higher one than a parameter. The full pipeline will merge windows and make bed files so post-process and merge the window output so I would use the bed outputs instead. Wouldn't telomeres always start at the start/end of a contig/read?

2. Sure, you can run the telomere finder on any fasta, including read sequences. Recent ONT reads are accurate enough, especially on human data. HiFi data is also OK.

**Reply by santiago-es** [reply]
Thanks for the reply! As for 1, for the p arm the start site is zero in T2T thankfully, but for the q arm I'm looking for basically the end of the subtelomere and start of the telomere, for the purposes of attempting to measure telomere length in my own reads after aligning to the T2T assembly, hence my question in 2.

I will attempt to do as you suggest and attempt to use the merged window output with a higher cutoff.
